composer.json

```diff
@@ -21,7 +21,7 @@
 		"typo3/cms-feedit": "*"
 	},
 	"replace": {
-		"frontend_editing": "self.version",
+		"friendsoftypo3/frontend_editing": "self.version",
```

Comment:
no, just remove that line! otherwise you replace your own version of the same package.

Reply:
Thanks. Should be fixed in next commit.